### PR TITLE
[BGD-719] certificate updates

### PR DESCRIPTION
**Pull request by ntfrnzn**
update: spark-operator helm chart
fix: restore rest-client for preinstall check
fix: certificate issuer settings for history server and EG

### CODE REVIEW

File: tide/manage.go [hunk 1]
```diff
@@ -130,9 +130,15 @@ func NewManager(log logr.Logger) (Manager, error) {
 		return nil, err
 	}
 
+	kubeConfig, err := buildRESTClientGetter(conf)
```
Comment (thorsteinnth) [inline tide/manage.go:133]: rename variable to e.g. `restClientGetter`?

**Review by skeggi — COMMENTED** [review]
LGTM
Agree with Thorsteinn's comment on the name change, the (build)getter is really just putting the struct together.

File: tide/manage.go (hunk 1 above)
Comment (arnarpall) [reply tide/manage.go:133]: 👍

**Review by arnarpall — COMMENTED** [review]
Looks good to me. Same comment as @thorsteinnth

File: tide/manage.go (hunk 1 above)
Comment (ntfrnzn) [reply tide/manage.go:133]: The reasoning is that "restClientGetter" is the name of the interface, "kubeconfig" is the name of the implementation type. But code is meant to be read, and all of you readers agree on the change :)

**Pull request merged**